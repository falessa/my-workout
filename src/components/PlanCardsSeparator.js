import React from 'react';
import { StyleSheet, View, Text } from 'react-native';

const styles = StyleSheet.create({
    separator: {
        height: 15,
    }
})

const PlanCardsSeparator = () => {
    return (
        <View style={styles.separator} />
    )
}

export default PlanCardsSeparator;