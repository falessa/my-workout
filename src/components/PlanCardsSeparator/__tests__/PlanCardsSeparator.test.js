import "react-native";
import React from "react";
import { create } from "react-test-renderer";
import PlanCardsSeparator from '../PlanCardsSeparator';

describe('PlanCardsSeparator tests', () => {
    const wrapper = create(<PlanCardsSeparator />)

    test('Render PlanCardsSeparator', () => {
        expect(wrapper).toMatchSnapshot()
    })
})