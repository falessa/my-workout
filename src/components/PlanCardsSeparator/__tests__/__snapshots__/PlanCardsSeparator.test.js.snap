// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlanCardsSeparator tests Render PlanCardsSeparator 1`] = `
<View
  style={
    {
      "height": 15,
    }
  }
/>
`;
